### PR TITLE
[WIP] interactive LUME background

### PR DESCRIPTION
**Pull request by trusktr**
Not ready yet. I had an hour today and got a scene hooked up and mouse events wired up (for parallax).

I got LUME imported as a global var just to get started, of course that won't be how we import it in the end. But then I had to install `lume` anyways to get some types in place.

### CODE REVIEW

File: src/pages/Home.tsx [hunk 1]
```diff
@@ -15,140 +15,168 @@ import productive from '../assets/icons/productive.svg';
 import wordmark from '../assets/wordmark.svg';
 import Footer from '../components/Footer';
 
-const Home: Component = () => (
-  <div class="flex flex-col">
-    <h1 class="sr-only">SolidJS homepage</h1>
+import { LumeHeaderBackground } from '../components/LumeHeaderBackground';
 
-    <header class="mx-3 rounded-br-3xl rounded-bl-3xl bg-gradient-to-r from-solid-light via-solid-medium to-solid-default text-white">
```
Comment (trusktr) [inline src/pages/Home.tsx:22]: The only changes in this file in in the `<header>` element. The rest is just a change of indentation (and hence maybe line formatting) due to adding logic in the Home component.

Perhaps we could separate the `<header>` into a separate component though.

File: src/pages/Home.tsx [hunk 2]
```diff
@@ -15,140 +15,168 @@ import productive from '../assets/icons/productive.svg';
...
+      <header
+        style="position: relative; overflow: hidden"
+        class="mx-3 rounded-br-3xl rounded-bl-3xl bg-gradient-to-r from-solid-light via-solid-medium to-solid-default text-white"
+        onpointermove={(event) => {
+          setMouseX(event.clientX);
+          setMouseY(event.clientY);
+        }}
+      >
+        {/* Place the LUME scene below the main header content. */}
+        <div style="position: absolute; width: 100%; height: 100%; z-index: 1;">
```
Comment (trusktr) [inline src/pages/Home.tsx:49]: Pardon my `style` attributes, I've no experience with Tailwind. If you don't mind letting me know what to write instead, I can change it (or just commit here).

Comment (modderme123) [reply src/pages/Home.tsx]: "absolute w-full h-full z-10" (I think)

**Comment by davedbase** [comment]
We can revisit this when we work on the 2.0 site. There's too many conflicts and so much to do rn. :-)

**Pull request closed without merging**